src/librustdoc/html/static/js/search.js

```diff
@@ -1710,22 +1704,15 @@ function initSearch(rawSearchIndex) {
         let crates = "";
         const crates_list = Object.keys(rawSearchIndex);
         if (crates_list.length > 1) {
-            crates = " in <select id=\"crate-search\"><option value=\"All crates\">" +
-                "All crates</option>";
+            crates = " in <select id=\"crate-search\"><option value=\"all crates\">" +
```

Comment:
Why this change?

Reply:
because "Result in all crates" reads better than "Result in All crates". Or do you mean the internal representation changing from "All crates" to "all crates", too? That just felt more consistent than just changing the label, but if there's a reason against it, it can of course be kept "All crates" internally.

Reply:
I just checked locally and I think it actually renders quite nicely locally.

Reply:
By the way, feel free to do your magic again for uploading an online versions of this change for others to check out.

Reply:
I'll after your next update. I really like this change. :)